frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -516,50 +519,56 @@ export const Chart = ({
               {chartTotal !== null && <DefaultCurrencyRotator tableRow={false} />}
             </span>
           </div>
-          {!showMobileTotalValue ? (
-            <PercentageDiff
-              hasDifference={!!hasDifference}
-              difference={difference}
-              title={diffSince}
-            />
-          ) :
-            <span className={styles.diffValue}>
-              {renderDate(toolTipTime * 1000, i18n.language, source)}
-            </span>
-          }
+          {!hideChartDetails && (
+            <>
+              {!showMobileTotalValue ? (
+                <PercentageDiff
+                  hasDifference={!!hasDifference}
+                  difference={difference}
+                  title={diffSince}
+                />
+              ) :
+                <span className={styles.diffValue}>
+                  {renderDate(toolTipTime * 1000, i18n.language, source)}
+                </span>
+              }
+            </>
+          )}
         </div>
-        {!isMobile && <Filters {...chartFiltersProps} />}
+        {!isMobile && !hideChartDetails && <Filters {...chartFiltersProps} />}
       </header>
-      <div className={styles.chartCanvas} style={{ minHeight: chartHeight }}>
-        {chartDataMissing ? (
-          <div className={styles.chartUpdatingMessage} style={{ height: chartHeight }}>
-            {t('chart.dataMissing')}
-          </div>
-        ) : hasData ? !chartIsUpToDate && (
-          <div className={styles.chartUpdatingMessage}>
-            {t('chart.dataOldTimestamp', { time: new Date(lastTimestamp).toLocaleString(i18n.language), })}
-          </div>
-        ) : noDataPlaceholder}
-        <div ref={ref} className={styles.invisible}></div>
-        <span
-          ref={refToolTip}
-          className={styles.tooltip}
-          style={{ left: toolTipLeft, top: toolTipTop }}
-          hidden={!toolTipVisible || isMobile}>
-          {toolTipValue !== undefined ? (
-            <span>
-              <h2 className={styles.toolTipValue}>
-                <Amount amount={toolTipValue} unit={chartFiat} />
-                <span className={styles.toolTipUnit}>{chartFiat}</span>
-              </h2>
-              <span className={styles.toolTipTime}>
-                {renderDate(toolTipTime * 1000, i18n.language, source)}
+      {(!hideChartDetails) ? (
```

Comment:
Nit: You could also do `{hideChartDetails || (...)}`

Reply:
Updated